_docs/management/container-to-container.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: management
+layout: docs
+sidenav: true
+redirect_from: 
+    - /docs/apps/container-to-container/
+title: Container-to-Container Networking
+---
+
+App developers can leverage cloud.gov's [container-to-container networking](https://docs.cloudfoundry.org/concepts/understand-cf-networking.html) feature to allow direct communication between their applications. This utilizes Cloud Foundry's [overlay network](https://docs.cloudfoundry.org/concepts/understand-cf-networking.html#overlay-network), which is not externally routable. Any traffic sent between containers on this network does not leave the overlay.
+
+## How to set up container-to-container networking
+
+To set up and allow traffic between your applications:
+
+1. Create an internal route to your application(s)
+```
+cf map-route APP apps.internal --hostname APP
+```
+2. Add a network policy to allow traffic from one application to another
+```
+cf add-network-policy SOURCE_APP DESTINATION_APP -s DESTINATION_SPACE_NAME -o DESTINATION_ORG_NAME --protocol (tcp | udp) --port RANGE
+```
```

Comment:
indent the backticks and code line so they'll be part of the ordered list and not restart the numbering
~~~suggestion
1. Create an internal route to your application(s)
   ```
   cf map-route APP apps.internal --hostname APP
   ```
2. Add a network policy to allow traffic from one application to another
   ```
   cf add-network-policy SOURCE_APP DESTINATION_APP -s DESTINATION_SPACE_NAME -o DESTINATION_ORG_NAME --protocol (tcp | udp) --port RANGE
   ```
~~~